support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -708,10 +708,25 @@ function CheckoutComponent({ geoId }: Props) {
 									}
 									amount={price}
 									currency={currency}
-									checkListData={productDescription.benefits.map((benefit) => ({
-										isChecked: true,
-										text: benefit.copy,
-									}))}
+									checkListData={[
+										...productDescription.benefits.map((benefit) => ({
+											isChecked: true,
+											text: benefit.copy,
+										})),
+										...(productDescription.missingBenefits ?? []).map(
+											(benefit) => ({
+												isChecked: false,
+												text: benefit.copy,
+												maybeGreyedOut: css`
```

Comment:
This is the current API - to use a `css` object. This could definitely be simplified, but I'm just reimplementing what we have to avoid breaking something in `PROD`.

Reply:
Looks neat to re-use `maybeGreyedOut` to me